src/main/java/com/iota/iri/network/protocol/Handshake.java

```diff
@@ -36,15 +43,14 @@
      */
     public static ByteBuffer createHandshakePacket(char ownSourcePort, byte[] ownByteEncodedCooAddress,
                                                    byte ownUsedMWM) {
-        short maxLength = ProtocolMessage.HANDSHAKE.getMaxLength();
-        final short payloadLengthBytes = (short) (maxLength - (maxLength - 60) + Protocol.SUPPORTED_PROTOCOL_VERSIONS.length);
-        ByteBuffer buf = ByteBuffer.allocate(ProtocolMessage.HEADER.getMaxLength() + payloadLengthBytes);
-        Protocol.addProtocolHeader(buf, ProtocolMessage.HANDSHAKE, payloadLengthBytes);
+        ByteBuffer buf = ByteBuffer.allocate(ProtocolMessage.HEADER.getMaxLength() + PAYLOAD_LENGTH_BYTES);
+        Protocol.addProtocolHeader(buf, ProtocolMessage.HANDSHAKE, PAYLOAD_LENGTH_BYTES);
         buf.putChar(ownSourcePort);
         buf.putLong(System.currentTimeMillis());
         buf.put(ownByteEncodedCooAddress);
         buf.put(ownUsedMWM);
         buf.put(Protocol.SUPPORTED_PROTOCOL_VERSIONS);
+        //update PAYLOAD_LENGTH_BYTES if changed
```

Comment:
Is this comment necessary?

Reply:
So the programmer won't forget to update this value if in the future another `buf.put` is added
you think it is too obvious?

Reply:
It's not "too" obvious. It's fine.